lib/transform-groups.js

```diff
@@ -1,8 +1,10 @@
 import StyleDictionary from 'style-dictionary';
 
+/** @typedef {import('style-dictionary').Named<import('style-dictionary').TransformGroup>} TransformGroup */
+
```

Comment:
typedef is analogous to `export type =`.

using it here to cut down on syntactic noise

should prolly just write ts sources, because `:` is always better than `/**@type{}*/`, but meh

Reply:
Oh gotcha!